src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+function createSuccess(message) {
+  const divSuccess = document.createElement('div');
+
+  divSuccess.textContent = message;
+  divSuccess.setAttribute('data-qa', 'notification');
+  divSuccess.classList.add('success');
+  document.body.appendChild(divSuccess);
+}
+
+function createError(message) {
+  const divError = document.createElement('div');
+
+  divError.textContent = message;
+  divError.setAttribute('data-qa', 'notification');
+  divError.classList.add('error');
+  document.body.appendChild(divError);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => createSuccess(message))
+  .catch((error) => createError(error));
```

Comment:
The `createError` function is called with an `Error` object, but it expects a string message. You should pass `error.message` instead of `error` to display the error message correctly.